mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItalianLinguisticAcceptabilityClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Itacola",
+        dataset={
+            "path": "gsarti/itacola",
+            "revision": "f8f98e5c4d3059cf1a00c8eb3d70aa271423f636",
+        },
+        description="An Italian Corpus of Linguistic Acceptability taken from linguistic literature with a binary annotation made by the original authors themselves.",
```

Comment:
re. score: during the development of the Scandinavian embedding benchmark I discovered that almost no model handles Linguistic acceptability tasks well. 

@Muennighoff, @imenelydiaker I am unsure if this is a shortcoming of current embedding models or not a desired properly (we want them to be semantic). I am in two camps about this.

Reply:
I wasn't exactly sure what to expect so I pretty much ran the linguistic acceptability evals already implemented and got the same results

Reply:
Can you run a naive Italian encoder on it as well (for the Scandinavian languages they seem to perform above chance).